src/assets/i18n/it.po

```diff
@@ -104,7 +104,7 @@ msgstr ""
 msgid "(Nessuna descruzione)"
 msgstr ""
 
-msgid "(Remove pool from database)"
+msgid "(Rimuovere pool dal database)"
```

Comment:
this still isn't correct, it needs to look like this:

msgid "(Remove pool from database)"
msgstr "(Rimuovere pool dal database)"

The translations uses the english string as the id to match to the translated string, it cannot find your translations if it does not know which string to match to